brauniger_iq.cc

```diff
@@ -56,13 +56,13 @@ inline state_t operator++(state_t& rs, int)
 
 static const int reqd_bytes[num_states] = { 6, 1, 2, 2, 25, 2, 2, 2, 2, 2, 2, 1, 2, 2, 2, 1 };
 
-static void rd_init(const char* fname)
+static void rd_init(const QString& fname)
 {
-  if (serial_handle = gbser_init(fname), NULL == serial_handle) {
-    fatal(MYNAME ": Can't open port '%s'\n", fname);
+  if (serial_handle = gbser_init(qPrintable(fname)), NULL == serial_handle) {
+    fatal(MYNAME ": Can't open port '%s'\n", qPrintable(fname));
```

Comment:
As an aside,  I've been slowly moving these to Fatal() << "prose" << fname but it's just complicated enough that I've not automated it.
I should probably come up with a clever way to get the module name by default.

https://github.com/gpsbabel/gpsbabel/blob/master/src/core/logging.h